tests/elements-regression/tests/elements-regression.test.js

```diff
@@ -11,18 +11,24 @@ const image2 = {
 	extension: 'jpg',
 };
 
+const image1 = {
+	title: 'image1',
+	extension: 'jpg',
+};
+
```

Comment:
Place this before image2 for better readability

Reply:
fixed